modules/sdk/project-templates/project-templates-rest-builder/src/test/java/com/liferay/project/templates/rest/builder/ProjectTemplatesRESTBuilderWorkspaceTest.java

```diff
@@ -112,6 +131,9 @@ public void testBuildTemplateRESTBuilderWorkspace() throws Exception {
 			temporaryFolder, "gradle", "gradleWS", _liferayVersion,
 			mavenExecutor);
 
+		writeGradlePropertiesInWorkspace(
+			gradleWorkspaceDir, "liferay.workspace.produc=" + _productKey);
```

Comment:
This looks like a typo.

Reply:
I will drop the chages and update rest builder tool to 1.0.220, then resend the pr